test/helpers.tsx

```diff
@@ -22,3 +23,46 @@ export function render(element: ReactElement) {
     }
   );
 }
+
+export function getButtonUtilityClass(slot: string) {
+  return generateUtilityClass('MuiButton', slot);
+}
+
+export const buttonClasses = generateUtilityClasses('MuiButton', [
```

Comment:
According to the source code, I think we don't need `@material-ui/unstyled` dependency. It's fairy enough to generate classes by creating a function like this
```ts
(prefix) => ['root', ...].reduce((classes, suffix) => ({ [suffix]: `${prefix}-${suffix}` }), {})
```

Reply:
I personally start writing selectors inline without the generator helper, because different components might have different suffixes. So, it's up to you. The main point is about the `@material-ui/unstyled` dependency.